pytorch_transformers/tokenization_distilbert.py

```diff
@@ -60,3 +60,21 @@ class DistilBertTokenizer(BertTokenizer):
     vocab_files_names = VOCAB_FILES_NAMES
     pretrained_vocab_files_map = PRETRAINED_VOCAB_FILES_MAP
     max_model_input_sizes = PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES
+
+    def add_special_tokens_single_sequence(self, token_ids):
```

Comment:
Are sure about the format for encoding DistilBert sequences and pair of sequences (I thought it used a classifier token as well). To be confirmed with @VictorSanh